swagger.json

```diff
@@ -959,6 +959,203 @@
                 }
             }
         },
+        "/fleet/unassigned_driving_segments": {
+            "get": {
+                "tags": [
+                    "Fleet", "Default"
+                ],
+                "summary": "/fleet/unassigned_driving_segments",
+                "description": "Get the unassigned driving segments for a specified range.",
+                "operationId": "get_fleet_unassigned_driving_segments",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/accessTokenParam"
+                    },
+                    {
+                        "name": "unassignedDrivingSegmentsParam",
+                        "required": true,
+                        "in": "body",
```

Comment:
We should use query parameters for these options and should prefer `start-ms` and `duration-ms`.

Reply:
@dsharrison I don't immediately see any query parameters that use skeleton case. Want to confirm that we're migrating to that paradigm?

Reply:
Correct, it looks like that change is still pending from the best practices. Let's stick with the previous best practice here unless @silverspace has more input.

Reply:
That's correct, we are not yet using spinal-case anywhere in our code base. When I was putting together our best practices doc, I saw that most REST guides advised using spinal-case for query parameters, and either camelCase or snake_case for JSON bodies. Since we use Golang, which defaults to camelCase, I've advocated for using camelCase in our JSON request/response bodies.

Reply:
With that said, I don't think we will preserve these existing endpoints in their current form for V2 of our APIs, so we can use an interim naming convention for the legacy APIs that aren't in our Golden API list. I am mostly concerned about the cohesive naming convention in all of our "v2" APIs.

Reply:
If we migrate to skeleton case, should we also update the access token parameter here to a skeleton cased version? Otherwise as it stands the parameters are `access_token`, `start-ms`, `end-ms`

Reply:
That's a good question, our APIs should definitely be internally consistent. We have two options:
1. Update the auth middleware to also accept `access-token`
2. Worry about spinal-case only for newer V2 APIs, which are part of https://paper.dropbox.com/doc/Golden-API-Routes--ARHywquzzMP3nl1Lnk8P1DzfAg-A6hbRcUq2NQFDuxtKb0si

I think it's ok to proceed with snake_case query parameters for now, since that's consistent with what all of our other APIs use.